src/LiveComponent/src/Controller/BatchActionController.php

```diff
@@ -38,6 +38,7 @@ public function __invoke(Request $request, MountedComponent $_mounted_component,
                 '_component_action_args' => $action['args'] ?? [],
                 '_mounted_component' => $_mounted_component,
                 '_live_component' => $serviceId,
+                '_routeScope' => $request->attributes->get('_routeScope'),
```

Comment:
What is `_routeScope`? I'm not familiar with it

Reply:
Whoops, thought this was something of symfony/router but I was wrong. Nvm then.